app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({self.name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = []
+    for person in people:
+        persons_list.append(Person(person["name"], person["age"]))
```

Comment:
Consider using list comprehension to create `persons_list` as recommended in the checklist. This will make the code more concise and efficient. For example: `persons_list = [Person(person['name'], person['age']) for person in people]`.